examples/auth/index.js

```diff
@@ -45,12 +47,17 @@ var users = {
 // when you create a user, generate a salt
 // and hash the password ('foobar' is the pass here)
 
-hash({ password: 'foobar' }, function (err, pass, salt, hash) {
-  if (err) throw err;
-  // store the salt & hash in the "db"
-  users.tj.salt = salt;
-  users.tj.hash = hash;
-});
+  //  ------------------ Old Way  ----------------- 
+// hash({ password: 'foobar' }, function (err, pass, salt, hash) {
+//   if (err) throw err;
+//   // store the salt & hash in the "db"
+//   users.tj.salt = salt;
+//   users.tj.hash = hash;
+// });
+
+//  ------------------ A better Way to Hash Password without passing any salt using Bcrypt.Js----------------- 
+var encryPassword = bcrypt.hashSync({ password: 'foobar' },10) //password Encrypter , here '10' is the salt used for encryption
```

Comment:
> here '10' is the salt used for encryption

I thought your PR said you don't need a salt? Can you vlarify this comment and/or PR description?

Reply:
You Need a Salt !
But We don't have to store the salt in database everytime for hashing the password

Reply:
Gotcha. So then the comment right above this seems misleading/confusing:

> A better Way to Hash Password without passing any salt using Bcrypt.Js

it says "without passing any salt", but this line's comment says "here '10' is the salt"

Reply:
> You Need a Salt !
> But We don't have to store the salt in database everytime for hashing the password

Salts should be unique for each user, otherwise their usability is reduced. With the same salt, identical passwords will still have identical hashes. Using random salts prevents this and makes cracking passwords harder. Constant salt almost completely neutralises the benefits of it.